### ISSUE TITLE
Set Page popup settings

### ISSUE THREAD

**Issue by cfredericksen** [saltcorn/saltcorn#2391]
**Is your feature request related to a problem? Please describe.**
My Page popup modal containing two views is too small.

**Describe the solution you'd like**
I have a page popup modal with two list views and it would be nice to be able to adjust the popup size for Pages like I can with views.

**Comment by gianlucafiore** [comment]
In the Edit section of the view, you have an accordion that grants the popup settings if the popup is displayed that way

![image](https://github.com/saltcorn/saltcorn/assets/50465150/e6a8fd70-b6d2-4e2b-9db4-61c99bc3cafc)

**Comment by cfredericksen** [comment]
> In the Edit section of the view, you have an accordion that grants the popup settings if the popup is displayed that way
> 
> ![image](https://github.com/saltcorn/saltcorn/assets/50465150/e6a8fd70-b6d2-4e2b-9db4-61c99bc3cafc)
> 

Yes, I have used the the pop-up settings in a view, works great. But when I add my two views to a page, I cannot adjust the pop-up width.
